src/Microsoft.Health.Dicom.SqlServer/Features/CustomTag/CustomTagLimit.cs

```diff
@@ -0,0 +1,41 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+using System.Collections.Generic;
+using Dicom;
+
+namespace Microsoft.Health.Dicom.SqlServer.Features.CustomTag
+{
+    /// <summary>
+    /// Limits on CustomTag feature.
+    /// </summary>
+    internal static class CustomTagLimit
```

Comment:
I don't really know about this class name. Perhaps something like `CustomTagCategories` with a method like `GetDataType`?

Reply:
I follow this class https://github.com/microsoft/dicom-server/blob/main/src/Microsoft.Health.Dicom.Core/Features/Query/QueryLimit.cs :)

Reply:
will leave as it is now.